lib/SILGen/SILGenLValue.cpp

```diff
@@ -497,8 +498,12 @@ SILValue UnenforcedAccess::beginAccess(SILGenFunction &SGF, SILLocation loc,
   if (!SGF.getOptions().VerifyExclusivity)
     return address;
 
-  SILValue base = findAccessedAddressBase(address);
-  if (!base || !isPossibleFormalAccessBase(base))
+  const AccessedStorage &storage = findAccessedStorage(address);
+  if (!storage) {
+    llvm::dbgs() << "Bad memory access source: " << address;
```

Comment:
`DEBUG(...)`?

Reply:
This is a thing I do to diagnose failures that need to be diagnosed "in the field". It's all guarded by a verification flag. Maybe I should be using llvm::errs() instead. DEBUG would make it useless because I need this information in the build log that I get out of certain automation that I have no way of reproducing. In general, it would save a lot of developers' time to do a better job leaving breadcrumbs behind when the compiler fails.